src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -1503,35 +1481,23 @@ private bool CanExecuteActiveCode()
             return _evaluator.CanExecuteCode(input);
         }
 
-        #endregion
+#endregion
 
-        #region Buffers, Spans and Prompts
-        private ITrackingSpan CreateStandardInputPrompt()
+#region Buffers, Spans and Prompts
+        private object CreateStandardInputPrompt()
         {
-            return CreateTrackingSpan(_standardInputPromptBuffer, string.Empty);
+            return string.Empty;
```

Comment:
It's probably too late, but it feels like the evaluator should have been responsible for determining what string to use here.